doc/source/whatsnew/v1.3.3.rst

```diff
@@ -21,6 +21,15 @@ Fixed regressions
 
 .. ---------------------------------------------------------------------------
 
+.. _whatsnew_133.performance:
+
+Performance improvements
+~~~~~~~~~~~~~~~~~~~~~~~~
+- Performance improvement for :meth:`DataFrame.__setitem__` with dataframes containing unique columns
```

Comment:
can you add the issue number

Reply:
not sure if I should mark this as the relevant issue https://github.com/pandas-dev/pandas/pull/39280#issuecomment-764753400.
Incase I am referring the PR (instead of issue), should I still use `:issue:`?

Reply:
> not sure if I should mark this as the relevant issue [#39280 (comment)](https://github.com/pandas-dev/pandas/pull/39280#issuecomment-764753400).

don't use that one as this issue is just a discussion after the PR was merged.

> Incase I am referring the PR (instead of issue), should I still use `:issue:`?

sure.

Reply:
amended